src/scf-release/jobs/acceptance-tests-brain/templates/run.erb

```diff
@@ -9,17 +9,13 @@ set -o errexit
 HELM_BIN_DIR=/var/vcap/packages/helm/bin
 HELM_VERSION="$(${HELM_BIN_DIR}/helm version | grep Server | sed -e 's/.*"v//' -e 's/".*//')"
 
-echo Packaged helm clients ...
-( cd ${HELM_BIN_DIR} ; find . -type f | sed -e 's/^/= /' )
 echo Helm server ${HELM_VERSION}
-echo Compare
```

Comment:
Dislike the removal of this, keeping would help future debugging. Still, can live with it.

Reply:
I added an informative statement inside the `if`. I'm not favorable of debugging statements, but informative statements that also help debugging I'm all up!